README.md

```diff
@@ -2,9 +2,89 @@
 
 Utility for generating Astro pages from `TypeDoc`.
 
+## Usage
+
+### Basic example
+
+Minimal config:
+
+```js
+// astro.config.mjs
+
+import { defineConfig } from 'astro/config'
+import { dirname, resolve } from 'node:path'
+import { fileURLToPath } from 'node:url'
+
+import initAstroTypedoc from 'astro-typedoc'
+
+const __dirname = dirname(fileURLToPath(import.meta.url))
+
+const astroTypedoc = await initAstroTypedoc({
+  baseUrl: '/docs/',
+  entryPoints: [resolve(__dirname, 'path/to/entry/point.ts')],
+  tsconfig: resolve(__dirname, 'path/to/tsconfig.json')
+})
+
+const typedocProject = await astroTypedoc.getReflections()
+
+await astroTypedoc.generateDocs(typedocProject, 'src/pages/docs')
+await astroTypedoc.generateNavigationJSON(
+  reflections,
+  resolve(__dirname, './src/')
+)
+
+// https://astro.build/config
+export default defineConfig({})
+```
+
+## API
+
+### `initAstroTypedoc({ baseUrl, entryPoints, tsconfig })`
+
+Creates `astroTypedoc` instance. Accepts config object as a parameter.
```

Comment:
It is not clear what is `astroTypedoc instance` and how it can be useful for me as a user.

Something like this could be better:

```
Creates a builder of pages and side-menu for your TypeDoc
```